schema.yml

```diff
@@ -0,0 +1,831 @@
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - $ref: '#/components/schemas/PersonDetailedBody'
+        400:
+          $ref: "#/components/responses/400BadRequestError"
+        401:
+          $ref: "#/components/responses/401UnauthorizedError"
+        403:
+          $ref: "#/components/responses/403ForbiddenError"
+        404:
+          $ref: "#/components/responses/404NotFoundError"
+        5XX:
+          $ref: "#/components/responses/5xxServerError"
+      tags:
+        - persons
+    post:
+      summary: Insert a new person into the database.
+      description: |
+        Insert a new person into the database
+      operationId: person_operations.post
+      parameters:
+      - in: path
+        name: id
+        required: true
+        schema:
+          title: Person Id
+          type: string
+      responses:
+        200:
+          description: Updated aggregate person information
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PersonAggregateBody'
+        400:
+          $ref: "#/components/responses/400BadRequestError"
+        401:
+          $ref: "#/components/responses/401UnauthorizedError"
+        403:
+          $ref: "#/components/responses/403ForbiddenError"
+        404:
+          $ref: "#/components/responses/404NotFoundError"
+        5XX:
+          $ref: "#/components/responses/5xxServerError"
+      tags:
+        - persons
+    delete:
+      summary: Delete a person from the database.
+      description: |
+        Delete a person from the database
+      operationId: person_operations.delete
+      parameters:
+      - in: path
+        name: id
+        required: true
+        schema:
+          title: Person Id
+          type: string
+      responses:
+        200:
+          description: Updated aggregate person information
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PersonAggregateBody'
+        400:
+          $ref: "#/components/responses/400BadRequestError"
+        401:
+          $ref: "#/components/responses/401UnauthorizedError"
+        403:
+          $ref: "#/components/responses/403ForbiddenError"
+        404:
+          $ref: "#/components/responses/404NotFoundError"
+        5XX:
+          $ref: "#/components/responses/5xxServerError"
+      tags:
+        - persons
+
+components:
+  requestBodies:
+    BeaconRequestBody:
+      content:
+        'application/json':
+          schema:
+            type: object
+            description: Schema for the Beacon request.
+            properties:
+              meta:
+                $ref: '#/components/schemas/BeaconRequestMeta'
+              query:
+                $ref: '#/components/schemas/BeaconQuery'
+            required:
+              - meta
+              - query
+  parameters:
```

Comment:
to make this more extensible and future-proof, let's not hard code parameters into the schema, but let the service return these dynamically via a 'search_parameters' or 'filtering_parameters' endpoint

Reply:
Done, I've copied over most of the Beacon stuff into beacon-schema.yaml which should let us avoid using a hardcoded parameters list